### DISCUSSION TITLE
Toggle / Escape Location Focus Keyboard Shortcut

### DISCUSSION THREAD

**Discussion by lowercaserace** [zen-browser/desktop#2733] [Ideas]
I'm running Zen in full screen, compact mode, pressing `⌘T` opens new tab and location is focused, after entering a query and pressing enter, navigation bar is auto hidden; nice. However, often times I need to copy or review the URL, pressing `⌘L` to open location, `⌘C` to copy link, but there is no shortcut to escape location focus -- the only way to do this is to mouse "click" somewhere in the body or switch to another tab and back which is not ideal.

e.g. navigating to example.com, pressing `⌘L` and no way to escape location focus:
![example](https://github.com/user-attachments/assets/f97b2d9c-0076-4558-940a-3d9545f88f1e)

<img width="2056" alt="image" src="https://github.com/user-attachments/assets/bce0df58-b601-445a-9b43-f35aeed08402">

It would be nice if `Open Location` shortcut can toggle focus. Pressing `⌘L` opens navigation bar and focuses on location (just like does natively) and pressing `⌘L` again releases focus and hides the navigation bar.

**Comment by MigueloAtla** [comment]
This is now possible in beta activating this setting in about:config "browser.urlbar.focusContentDocumentOnEsc"

**Reply by lowercaserace** [reply]
You're the best! Thank you! -- and happy New Year!
